aptos-move/e2e-move-tests/src/tests/move_feature_gating.rs

```diff
@@ -76,13 +76,8 @@ fn resource_access_control(enabled: Vec<FeatureFlag>, disabled: Vec<FeatureFlag>
     }
 }
 
-#[rstest(enabled, disabled,
-    case(vec![], vec![FeatureFlag::ENABLE_FUNCTION_VALUES]),
```

Comment:
this is removed because aptos framework is compiled using 2.2, which will cause VM error in `publish_package_with_options` below when `ENABLE_FUNCTION_VALUES` is disabled.